w3c_validators.gemspec

```
@@ -16,6 +16,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'nokogiri', '~> 1.6'
   gem.add_dependency 'json', '>= 1.8'
+  gem.add_dependency 'rexml'
```

To keep consistency I would prefer that we add the version behind
```suggestion
  gem.add_dependency 'rexml', '~> 3.2é
```